EnglishFilter/sections/whitelist.txt

```diff
@@ -144,6 +144,9 @@ referatwork.ru#@##adbox
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36735
 @@||animeland.us/pa.js
 @@||animeland.us/bg.jpg
+! https://github.com/AdguardTeam/AdguardFilters/issues/37260
+@@||sharethrough.com^$domain=looper.com
+@@||connatix.com^$domain=looper.com
```

Comment:
Maybe it would be better to make these rules more specific?
For example:
```
@@||sharethrough.com/*placement$domain=looper.com
@@||native.sharethrough.com/assets/sfp.js$domain=looper.com

@@||cdn*.connatix.com/*min/connatix.renderer.infeed.min$domain=looper.com
@@||connatix.com/*?callback=$domain=looper.com
@@||i.connatix.com/*/connatix-uploads/$domain=looper.com
```

Reply:
done